Fix extra events being emitted from the color slider

paulkaplan opened the pull request:
### Resolves

_What Github issue does this resolve (please include link)?_

Related to LLK/scratch-vm#1695, we were getting echo events from setting the initial slider values. 

Move the call to set those values above the callbacks being bound to avoid this.

paulkaplan committed:
Fix extra events being emitted from the color slider

kchadha reviewed (changes requested):
Hmmm for some reason this change makes it so that the slider thumbs are not initialized to the correct positions when the field editor is opened...

paulkaplan committed:
Move binding callbacks below showing dropdown div so sliders update

They were not updating because the component does not update if it is not visible in the DOM

kchadha reviewed (approved):
LGTM

kchadha commented:
Merging since I need these changes for the PR I am about to submit.
